apiserver/facades/controller/cleaner/cleaner_test.go

```diff
@@ -166,3 +170,17 @@ func (st *mockState) Cleanup(_ context.Context, _ objectstore.ObjectStore, mr st
 	st.MethodCall(st, "Cleanup", mr, ar)
 	return st.NextErr()
 }
+
+type stubResourceStoreGetter struct{}
```

Comment:
__nit:__ could this be called something other than `stub`, like `dummy` or `noop`. Stub to me means something specific in the context of Juju tests where we use the `testing.Stub` and check the stub calls. (Ignore this if this is actually one of those)